saleor/core/utils/filters.py

```diff
@@ -1,6 +1,6 @@
 def get_sort_by_choices(filter_set):
     return [(choice[0], choice[1].lower()) for choice in
-            filter_set.filters['sort_by'].field.choices[1::2]]
+            filter_set.filters['sort_by'].field.choices.choices[0::2]]
```

Comment:
In version `1.1.0` of `django-filters` statement `filter_set.filters['sort_by'].field.choices` returns `ChoiceIterator` object, not just a `list`.
I tried to make the function the same output as before.
@elwoodxblues @koradon Can you please take a look if this is a correct way to adapt to new version?

Reply:
Wouldn't `list(filter_set.filters['sort_by'].field.choices)[0::2]` be more readable? The `.choices` field seems to be an implementation detail. Also I'd love to see a comment explain why we only select every other option and why indexing it at either `0` or `1` makes sense.

Reply:
@patrys Sure, this implementation seems to be more readable. In this case however, indexes `[1::2]` has to be taken to display proper filtering options... Strange indeed.
@koradon I see you have implemented this function some time ago, can you please shed some light on how it works? I mean, those indexes "just works", but is there any reason behind choosing particular ones?

Reply:
This is done this way because sort_by OrderingFilter generates a list of choices from SORT_BY_FIELDS. In our case it will be sth like this
 `['no ordering', 'price', '-price', ...]` 
and because of our custom rendering on category/index.html page we need only those positive values. Maybe it can be done simpler today. This is one of the things we have to do to have the same look as the React version of the storefront.

Reply:
@dominik-zeglen was also working on this. Maybe he remembers more.

Reply:
Maybe it would be more elegant to pass a list of field names that should be picked from the `field.choices` list as available sorting options? I would also prefer using `list(...)` instead of `choices.choices`.